docs/docs/authentication/google-sso.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: default
+title: Google
+nav_order: 1
+parent: Authentication
+description: >
+  This page contains instructions on setting up Google
+  as the SSO provider for Firezone.
+---
+---
+
+Firezone supports Single Sign-On (SSO) through Google Workspace.
+After successfully configuring SSO with Firezone, users will be prompted to sign
+in with their Google credentials to in the Firezone portal to authenticate VPN
+sessions, and download device configuration files.
+
+![Firezone Google SSO Login](https://user-images.githubusercontent.com/52545545/156853456-1ab3f041-1002-4c79-a266-82acb5802890.gif){:width="600"}
+
+To set up SSO, follow the steps below:
+
+## Step 1 - Configure OAuth Consent Screen
+
+If this is the first time you are creating a new OAuth client ID, you will
+be asked to configure a consent screen.
+
```

Comment:
Probably want to add,

**IMPORTANT**: Select `Internal`...

Otherwise, anyone with a Google account worldwide will be able to log in to this Firezone instance (technically not our fault, but shouldn't make it this easy to get compromised).